pkg/asset/ignition/bootstrap/common.go

```diff
@@ -278,6 +280,18 @@ func (a *Common) getTemplateData(dependencies asset.Parents, bootstrapInPlace bo
 		logrus.Warnf("Found override for Cluster Profile: %q", cp)
 		clusterProfile = cp
 	}
+
+	// Set Tear-down delay for platform specific
+	var tearDownDelay string
+	switch installConfig.Config.Platform.Name() {
+	case alibabacloud.Name:
+		// tear-down set to let kube-apiserver rolls out and avoid loopback CLB limitation
+		// BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035757
+		tearDownDelay = "10m"
+	default:
+		tearDownDelay = "0"
+	}
```

Comment:
Let's make the happy path where we don't have to set the delay more prominent.
```suggestion
	tearDownDelay := "0"
	if installConfig.Config.Platform.Name() == alibabacloud.Name {
		// tear-down set to let kube-apiserver rolls out and avoid loopback CLB limitation
		// BZ https://bugzilla.redhat.com/show_bug.cgi?id=2035757
		tearDownDelay = "10m"
	}
```